### PR TITLE
docs: Add Phoenix Llamaindex RAG Demo notebook + chunks + questions

### LINKED ISSUE #4004: [onboarding] cohesive demo data
Create cohesive demo data that is based on real-world scenarios and evaluations. This should focus on:

- [ ] #4009
- [ ] Agents with Tool Calling

### PR DESCRIPTION
**Pull request by saikolasani**
resolves #4004

### CODE REVIEW

**Comment by mikeldking** [comment]
nit: can we rename to `demo_llama_index` just to keep track of demo data creation notebooks?

File: tutorials/internal/PhoenixLlamaIndexRAGDemo/llama_index_documents.json [hunk 1]
```diff
@@ -0,0 +1,2396 @@
+[
```
Comment (mikeldking) [inline tutorials/internal/PhoenixLlamaIndexRAGDemo/llama_index_documents.json:1]: we should git ignore this file

File: tutorials/internal/PhoenixLlamaIndexRAGDemo/phoenix_llama_index_documents.json [hunk 2]
```diff
@@ -0,0 +1,2396 @@
+[
```
Comment (mikeldking) [inline tutorials/internal/PhoenixLlamaIndexRAGDemo/phoenix_llama_index_documents.json:1]: also gitignore this

**Review by mikeldking — COMMENTED** [review]
I think this is pretty close to just merging. Let's remove the artifacts of the chunking for now

**Comment by cjunkin** [comment]
resolves #4240

File: .gitignore [hunk 3]
```diff
@@ -25,3 +25,6 @@ coverage.xml
 
 # devtools
 pyrightconfig.json
+
+# Demo data
+tutorials/internal/demo_llama_index/*.json
```
Comment (mikeldking) [inline .gitignore:30]: maybe just put this in a .gitignore in demo_llama_index so it's self contained

File: src/phoenix/trace/fixtures.py [hunk 4]
```diff
@@ -82,6 +82,26 @@ class TracesFixture:
     dataset_fixtures: Iterable[DatasetFixture] = ()
 
 
+demo_llama_index_cohesive_rag_fixture = TracesFixture(
```
Comment (mikeldking) [inline src/phoenix/trace/fixtures.py:85]:
```suggestion
demo_llama_index__rag_fixture = TracesFixture(
```

File: src/phoenix/trace/fixtures.py [hunk 5]
```diff
@@ -82,6 +82,26 @@ class TracesFixture:
...
+    name="demo_llama_index_cohesive_rag",
```
Comment (mikeldking) [inline src/phoenix/trace/fixtures.py:86]:
```suggestion
    name="demo_llama_index_rag",
```

File: src/phoenix/trace/fixtures.py [hunk 6]
```diff
@@ -82,6 +82,26 @@ class TracesFixture:
...
+    description="Cohesive RAG data that is based on real-world scenarios and evaluations.",
```
Comment (mikeldking) [inline src/phoenix/trace/fixtures.py:87]:
```suggestion
    description="Traces and evaluations of a RAG chatbot using LlamaIndex.",
```

File: src/phoenix/trace/fixtures.py [hunk 7]
```diff
@@ -82,6 +82,26 @@ class TracesFixture:
...
+    file_name="llama_index_cohesive_rag.parquet",
+    evaluation_fixtures=(
```
[the remark this answers is not in the document]
Comment (mikeldking) [reply src/phoenix/trace/fixtures.py:89]: were you able to spin up a phoenix?

**Review by mikeldking — APPROVED** [review]

**Comment by cjunkin** [comment]
Demo traces available on hosted phoenix

image

**Comment by cjunkin** [comment]
However, seems like document relevance eval fixtures have some formatting problems
image

**Pull request merged**